src/raylib.h

```diff
@@ -304,9 +304,10 @@ typedef struct Font {
 typedef struct Camera3D {
     Vector3 position;       // Camera position
     Vector3 target;         // Camera target it looks-at
-    Vector3 up;             // Camera up vector (rotation over its axis)
+    Vector3 up;             // Camera up vector (rotation around the view axis)
     float fovy;             // Camera field-of-view apperture in Y (degrees) in perspective, used as near plane width in orthographic
     int projection;         // Camera projection: CAMERA_PERSPECTIVE or CAMERA_ORTHOGRAPHIC
+    int swingCounter;       // Camera view-bobbing. Set to 0 to deactivate.
```

Comment:
while I understand moving this here stores the data relative to the camera and not in a global, I feel that people are not going to understand setting a count to 0 as disable (even with the comment). It may be worth having a viewBob enabled Boolean just to ease confusion.